dizoo/evogym/config/bridgewalker_ddpg_config.py

```diff
@@ -9,7 +9,7 @@
         collector_env_num=1,
         evaluator_env_num=1,
         n_evaluator_episode=1,
-        stop_value=1,
+        stop_value=10,
```

Comment:
why 10 here

Reply:
Experiment show that it could be far more than 1. After 12h training in ddpg, it approximate 4.
According to the evogym code, it may be 10 or so, similar to walker environment.